src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs

```diff
@@ -11,7 +11,7 @@ namespace System.Globalization.Tests
     internal static class NumberFormatInfoData
     {
         private static readonly bool s_isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
-        private static readonly int s_WindowsVersion = DateTimeFormatInfoData.GetWindowsVersion();
+        private static readonly int s_WindowsVersion = PlatformDetection.WindowsVersion;
         private static readonly bool s_isOSX = RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
         private static readonly Version s_OSXKernelVersion = GetOSXKernelVersion();
```

Comment:
I didn't want to change NumberFormatInfo too much in this PR, but beware, I've got a PR coming that's gonna cleanup a whole bunch of stuff in NFI tests